packages/callout/src/Callout.tsx

```diff
@@ -235,17 +113,37 @@ function Callout({
       </div>
       <div className={bodyStyle}>
         {title && (
-          <Subtitle as="h3" className={cx(titleStyle, fontSet[baseFontSize])}>
+          <Subtitle
+            as="h3"
+            className={cx(titleStyle, bodyTypeScaleStyles[baseFontSize])}
+          >
             {title}
           </Subtitle>
         )}
-        <div className={fontSet[baseFontSize]}>{contents}</div>
+        <div
+          className={cx(
+            bodyTypeScaleStyles[baseFontSize],
+            contentStyles,
+            css`
+              a {
+                color: ${colorSet.link.color};
+                &:hover {
+                  color: ${colorSet.link.hoverColor};
+                }
+              }
+            `,
```

Comment:
Could we break this out into `styles.ts`?

Reply:
yeah

Reply:
For this component it makes sense to keep it this way to be consistent with the other theme/variant styles.